web/src/components/BlogCard/BlogCard.tsx

```diff
@@ -30,7 +20,7 @@ const BlogCard = ({ post }: BlogCardProps) => {
   return (
     <article className="rounded-[4px] border-1 border-maiTai p-7 pb-5">
       <p className="mb-2 text-sm font-bold uppercase text-maiTai">
-        {prettifyDate(post.publishedAt)}
+        {prettifyDate(post.publishedAt.toString())}
```

Comment:
I'll follow up with a change to that function to not do string->date->string. I didn't want to add it to this PR.